test/cmdlineTests/optimizer_inliner_dynamic_reference/output

```diff
@@ -164,29 +164,35 @@ sub_0: assembly {
       iszero
       tag_28
       jumpi
-      shl(0xe0, 0x4e487b71)
-        /* "#utility.yul":293:326   */
-      dup2
+        /* "#utility.yul":312:322   */
+      0x4e487b71
+        /* "#utility.yul":307:310   */
+      0xe0
+        /* "#utility.yul":303:323   */
+      shl
```

Comment:
Curious how does the assembly output printer decides when this can be combine into `shl(0xe0, 0x4e487b71)` and when it can't?

Reply:
It has to be split if the source location changes, for example.